osdk/tools/docker/tdx/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: MPL-2.0
+
+ARG BASE_VERSION
+FROM asterinas/osdk:${BASE_VERSION} AS build-base
+
+# Fetch and install QEMU from the intel-staging/qemu-tdx source
+FROM build-base AS build-qemu-tdx
+
+RUN apt update && apt-get install -y --no-install-recommends \
+    libgcrypt-dev   `# optional build dependency` \
+    libglib2.0-dev  `# build dependency` \
+    libpixman-1-dev `# build dependency` \
+    libusb-dev      `# optional build dependency` \
+    meson \
+    ninja-build
+RUN apt clean && rm -rf /var/lib/apt/lists/*
+
+FROM build-qemu-tdx AS qemu-tdx
+
+WORKDIR /root
+RUN git clone -b tdx-qemu-upstream-2024.02.29-v8.2.0 https://github.com/intel-staging/qemu-tdx.git
+WORKDIR /root/qemu-tdx
+COPY tools/docker/tdx/tdx_qemu.patch /root/qemu-tdx
```

Comment:
I don't think the path `tools/docker/tdx/tdx_qemu.patch` is valid. The `osdk/tools/docker/README.md` file says that the Docker build command is executed at the project root. So the Docker build context is the project root. And the patch file has been moved to `osdk/`. So there is no `tools/docker/tdx/tdx_qemu.patch`, but only `osdk/tools/docker/tdx/tdx_qemu.patch`.

This leads to my second question: what about `tools/docker/tdx/Dockerfile`? It is not deleted by this PR but refers to the renamed patch file. So the `Dockerfile` now becomes broken.

Reply:
Sorry, missed updating the patch path.